workflows/create_meeting.ts

```diff
@@ -36,9 +36,14 @@ const SetupWorkflowForm = CreateMeeting.addStep(
         },
         {
           name: "date",
-          title: "Select a time to send the meeting",
+          title: "Select a time to schedule the meeting",
           type: Schema.slack.types.timestamp,
         },
+        {
+          name: "name",
```

Comment:
feel like the UI will look better if "name" was above "date"
well personally i think "name" should be first in the dialog

Reply:
Good call, I like this better!
![image](https://github.com/stmcdonald-vt/convo-crafters/assets/89744909/2f49764a-4a27-491f-a10c-bea189320fdd)